BorrowToOwn.Services/Implementations/ProductService.cs

```diff
@@ -88,8 +88,7 @@ public async Task<ProductResponseObject> AddProductAsync(ProductRequestObject pr
         public async Task<ProductResponseObject> GetProductAsync(long id)
         {
             var product = await _productRepo.GetProductAsync(id);
-            if (product == null) return null;
-            return _mapper.Map<ProductResponseObject>(product);
+            return product == null ? null : _mapper.Map<ProductResponseObject>(product);
```

Comment:
returning null from a method...... i understand what u are doing.... but.....i'm not sure what the solution is, so i'll pass